test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetConsoleTestExtension.cs

```diff
@@ -52,8 +52,14 @@ public void UpdatePackageFromPMC(string packageId, string version, string source
         public void UpdatePackageFromPMCWithConstraints(string packageId, bool minor, bool patch)
```

Comment:
it would be more generic if you just overloaded this function with a string argument that accept a bunch of arguments, and then you can pass a value like "-ToHighestPatch -ToHighestMinor" from the calling function